css/sekcje/howitworks.css

```diff
@@ -0,0 +1,74 @@
+/* ------------------------------MOBILE */
+
+.howitworks {
+    padding-top: 80px;
+    margin: 0 auto;
+}
+
+.howitworks-title {
+    margin-bottom: 14px;
+}
+.howitworks-text {
+    text-align: center;
+    margin-bottom: 44px;
+}
+
+.howitworks-fieldset {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    margin: 0 auto;
+    align-items: center;
+}
+
+.howitworks-field {
+    width: 335px;
+    height: 112px;
+    border-radius: 20px;
+    background-color: var(--clr-white);
+    display: flex;
+    align-items: center;
+    padding: 20px;
+}
+
+.howitworks-steps-btn {
+    background-color: var(--clr-red);
+    width: 136px;
+    height: 48px;
+    border-radius: 200px;
```

Comment:
Wartość jest bardzo duża. Na live serwer to dobrze wygląda?

Reply:
tak, wygląda jak w Figmie, identycznie